packages/renderer/src/docs/home.vue

```diff
@@ -1,63 +1,355 @@
+                              </q-item-section>                         
+                          </q-item>
+                          <q-item>
+                              <q-item-section>                                
+                                Protocols can be added to individual studies or assays, respectively. 
+                              </q-item-section>                              
+                          </q-item>
+                          
+                          
+                          <q-separator inset/>
+                          <br>
+                          <div class="text-subtitle1 text-weight-bold">Assay datasets</div>
+                          <q-item>
+                              <q-item-section>
+                                <div class="q-tree q-tree--dense"  role="tree" style="padding: 1em;"><div class="q-tree__node-header relative-position row no-wrap items-center " tabindex="0" role="treeitem" aria-expanded="true"><div class="q-focus-helper" tabindex="-1"></div><i class="q-icon notranslate material-icons q-tree__arrow q-tree__arrow--rotate">play_arrow</i><!----><div class="q-tree__node-header-content col row no-wrap items-center"><!--v-if--><div style="flex: 100 1 0%; white-space: nowrap;">dataset</div><!--v-if--></div></div><div class="q-tree__node-collapsible" style=""><!----><div class="q-tree__children" role="group"><div class="q-tree__node relative-position q-tree__node--child"><div class="q-tree__node-header relative-position row no-wrap items-center " tabindex="0" role="treeitem"><div class="q-focus-helper" tabindex="-1"></div><!----><!----><div class="q-tree__node-header-content col row no-wrap items-center"><i class="q-icon notranslate material-icons" style="padding: 0px 0.2em 0px 0px;">add_box</i><div style="flex: 100 1 0%; white-space: nowrap;">Import Dataset Files</div><!--v-if--></div></div><!----></div><div class="q-tree__node relative-position q-tree__node--child"><div class="q-tree__node-header relative-position row no-wrap items-center " tabindex="0" role="treeitem"><div class="q-focus-helper" tabindex="-1"></div><!----><!----><div class="q-tree__node-header-content col row no-wrap items-center"><i class="q-icon notranslate material-icons" style="padding: 0px 0.2em 0px 0px;">add_box</i><div style="flex: 100 1 0%; white-space: nowrap;">Import Dataset Folders</div><!--v-if--></div></div><!----></div></div></div></div>
+                              </q-item-section>                         
+                          </q-item>
+                          <q-item>
+                              <q-item-section>                                
+                                Dataset files or folders can imported to individual assays.
```

Comment:
This help menu has a awkward issue, that once you clicked on any button/navigation you cannot reach this help menu again. At least as far as i tested it. There is not button/ page that returns the ARCitect to the init state in which this is shown.